_episodes/04b-managing-containers.md

```diff
@@ -110,8 +110,9 @@ $ docker container rm 9c698655416a
 ~~~
 {: .output}
 
-**Be careful** with this command.
-If you have containers you may want to reconnect to, you should not use this command.
+**Be careful** with this command as deleting the container is **forever**.
```

Comment:
I might be reading this out of context but looking at the complete file, there is not actually a mention that we're going to run the `docker container prune` command as an alternative to `docker container rm`. Since this is under the section "How do I remove an exited container", I wonder if a little additional context is useful here to highlight that this is not actually a direct alternative to `docker container rm` since it removes _all_ stopped containers? Perhaps something like:

```suggestion
An alternative option for deleting exited containers is the `docker container
prune` command. Note that this command doesn't accept a container ID as an
option because it deletes ALL exited containers!
**Be careful** with this command as deleting the container is **forever**.
```